lib/generated_definitions/nz.rb

```diff
@@ -2,7 +2,7 @@
 module Holidays
   # This file is generated by the Ruby Holidays gem.
   #
-  # Definitions loaded: definitions/nz.yaml
+  # Definitions loaded: ../definitions/nz.yaml
```

Comment:
Make sure to remove this change here from our local file trick

Reply:
Good pick up. Thanks.

Reply:
btw, you'll probably have to change the `payaus` commit hash to reference the one 'on-merge'